doc/conf.py

```diff
@@ -132,7 +136,11 @@ def __getattr__(cls, name):
 #    'backreferences_dir': False
 # }
 
+read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+
+if read_the_docs_build:
```

Comment:
why only on readthedocs? shouldn't users who want to build it locally also have the models documentation?
I would rather wrap it to prevent errors if doxygen is not installed

Reply:
Yes, I think these two lines are obsolete.